concordium-smart-contract-testing/src/impls.rs

```diff
@@ -1301,6 +1734,22 @@ pub(crate) fn contract_events_from_logs(logs: v0::Logs) -> Vec<ContractEvent> {
     logs.logs.into_iter().map(ContractEvent::from).collect()
 }
 
+impl From<concordium_rust_sdk::endpoints::QueryError> for ExternalNodeError {
+    fn from(_: concordium_rust_sdk::endpoints::QueryError) -> Self { Self::QueryError }
+}
+
+impl From<concordium_rust_sdk::endpoints::Error> for SetupExternalNodeError {
+    fn from(_: concordium_rust_sdk::endpoints::Error) -> Self { Self::CannotConnect }
+}
+
+impl From<ExternalNodeNotConfigured> for ExternalNodeError {
+    fn from(_: ExternalNodeNotConfigured) -> Self { Self::NotConfigured }
+}
+
+impl From<ExchangeRateError> for ChainBuilderError {
+    fn from(_: ExchangeRateError) -> Self { Self::ExchangeRateError }
+}
```

Comment:
Why are we forgetting the inner error?

I think those generally provide useful information that can be used to debug what went wrong.

Reply:
Changed now.